about-pses/example-pse.md

```diff
@@ -1,90 +1,372 @@
 
-Imagine working on software that deals with restaurant reviews. Create a function named `get_highest_rated` that is responsible for finding the highest-rated restaurant. This function should take in a list of dictionaries called `restaurants` as a parameter. Each dictionary represents the data associated with a restaurant, including its rating. This function should have a return value of the restaurant with the highest rating.
+Create a function named `winner` that takes two arguments, the move for `player_1` and `player_2` (`"rock"`, `"paper"`, or `"scissors"`), and returns the winner (`"Player 1 wins!"`, `"Player 2 wins!"`, or `"It's a tie"`).
 
-Answer the following prompts:
+## Example Input/Output
 
-1. List five or more questions whose answers would clarify the problem statement
-2. List two sets of example arguments and the expected return value for these arguments
-3. Divide the project prompt into at least two different sub-problems
-4. Pick one of those problems from #3 and write a numbered list explaining how to solve that problem without using full lines of code
-5. Write the pseudocode or code solution for your answer for #4
+| Input (Argument of the function) | Expected Output (Return value of the function) | Explanation|
+|--|--|--|
+|player_1 = `"rock"` <br> player_2 = `"scissors"`| `"Player 1 wins!"`|`"rock"` beats `"scissors"`|
+|player_1 = `"rock"` <br> player_2 = `"rock"`| `"It's a tie!"`|`player_1` and `player_2` played the same move|
 
-## Example Answers
+### Complete Rock, Paper, Scissors Ruleset
+| input (first argument)| input (second argument) | output |
+|--|--|--|
+|**player_1** |**player_2**|**return value**|
+|`"rock"`|`"rock"`|`"It's a tie!"`|
+|`"paper"`|`"paper"`|`"It's a tie!"`|
+|`"scissors"`|`"scissors"`|`"It's a tie!"`|
+|`"rock"`|`"paper"`|`"Player 2 wins!"`|
+|`"rock"`|`"scissors"`|`"Player 1 wins!"`|
+|`"paper"`|`"rock"`|`"Player 1 wins!"`|
+|`"paper"`|`"scissors"`|`"Player 2 wins!"`|
+|`"scissors"`|`"rock"`|`"Player 2 wins!"`|
+|`"scissors"`|`"paper"`|`"Player 1 wins!"`| 
 
-### #1 Five or More Clarifying Questions
+## Prompts
 
-1.  Should I return the restaurant's name, or the dictionary that represents the restaurant data, or the rating?
-1.  What is the name of the key in each dictionary whose value is the rating?
-1.  What kind of data-type is rating stored in?
-1.  What should happen if there's more than one highest-rated restaurant?
-1.  Is the restaurant's name included in this dictionary?
-1.  What else is included in each dictionary?
-1.  Will there ever be zero restaurants? What should happen?
-1.  What should I do if there is no rating?
+<!-- Question 1 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: paragraph
+* id: 164a62ea-f936-4c6a-8b74-6d8248ba9c9d
+* title: Ask Clarifying Questions
+* topics: pse
+##### !question
 
-### #2 Two Example Inputs/Outputs
+List five or more questions whose answers would clarify the problem statement
```

Comment:
What do you all think about lowering this requirement? Personally, I believe that this has forced students in the past to make up questions to ask for the sake of hitting the 5 mark instead of coming up with solid clarifying questions that make sense in the context.

What if we did 3 or more?

Reply:
Agree! Auberon and I had a similar conversation earlier, and I think lowering the requirement might be the right solution!

Reply:
so maybe: 

```suggestion
List three or more questions whose answers would clarify the problem statement
```